index.ts

```diff
@@ -261,9 +261,8 @@ wss.on("request", async function (request) {
         msg = JSON.stringify({ type: "reset" });
         break;
       case "kick":
-        let kickconnection = room.connections.filter((conn) => conn.id == data.id)[0]
-          .connection;
-        kickconnection.close();
+        room.connections.filter((conn) => conn.id == data.id)
+          .map((conn) => conn.connection.close());
```

Comment:
```suggestion
          .forEach((conn) => conn.connection.close());
￼```

may be more appropriate since the return values are not used - but regarding the function it does not make a difference.

Reply:
I did not test this but I read that map is more performant. There are several cases in our code where it would be more semantic to use `forEach`. IMO we should decide which is more important (performance or semantics) and refactor once.

Reply:
hm, indeed. I wasn't aware of this performance advantage